metadata-ingestion/src/datahub/ingestion/source/sql/hive.py

```diff
@@ -35,6 +38,10 @@ class HiveConfig(BasicSQLAlchemyConfig):
     # Disabling views helps us prevent this duplication.
     include_views = False
 
+    @validator("host_port")
+    def remove_trailing_slash(cls, v):
```

Comment:
Let's update name of this method to reflect what it does.

Reply:
done